src/libs/actions/MemoryOnlyKeys/MemoryOnlyKeys.js

```diff
@@ -0,0 +1,18 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../../ONYXKEYS';
+
+const memoryOnlyKeys = [ONYXKEYS.COLLECTION.REPORT, ONYXKEYS.COLLECTION.POLICY, ONYXKEYS.PERSONAL_DETAILS_LIST];
+
+const enable = () => {
+    console.log('Turning on memory only keys');
```

Comment:
accrued lint error.
As you can see, there is `Unexpected console statement` error